src/leds.rs

```diff
@@ -149,3 +149,110 @@ impl Leds {
         ]
     }
 }
+
+impl<'a> IntoIterator for &'a mut Leds {
+    type Item = &'a mut Led;
+    type IntoIter = LedsMutIterator<'a>;
+
+    fn into_iter(self) -> Self::IntoIter {
+        self.iter_mut()
+    }
+}
+
+const ITERATOR_SIZE: usize = 8;
+
+pub struct LedsMutIterator<'a> {
```

Comment:
Does this need to be public?
I'm not sure.

Reply:
Needs a doc comment if so.

Reply:
Does need to be public, but I don't actually think it needs a doc comment upon further reflection.